frontend/packages/console-plugin-shared/src/components/OverviewDetailItem.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+import * as classNames from 'classnames';
+
+import './OverviewDetailItem.css';
+
+export type OverviewDetailItemProps = {
+  title: string;
+  children: React.ReactNode;
+  isLoading?: boolean;
+  valueClassName?: string;
+
+  error?: boolean;
+  /** Text for error === true, use "Not available" as a fallback */
+  errorMessage?: string;
```

Comment:
Thinking about a way to enforce that in case of `error !== undefined`. Any idea?

What about changing the `error` to just hold the message, so `error?: string`?

Reply:
I agree. `error?: string` is better.

Reply:
done